app/forms/submissions/trial/step03_content_form.rb

```diff
@@ -2,8 +2,27 @@ module Submissions
   module Trial
     class Step03ContentForm < PlantTrialForm
       property :lines_or_varieties
-      property :technical_replicate_numbers
-      property :design_form_names
+      collection :technical_replicate_numbers
+      collection :design_factor_names
+
+      validate do
+        all_valid = technical_replicate_numbers.all? do |number|
+          number.to_i.to_s == number && number.to_i > 0
+        end
+        errors.add(:technical_replicate_numbers, :invalid) unless all_valid
+      end
+
+      def technical_replicate_numbers
+        super || []
+      end
+
+      def technical_replicate_nummbers=(numbers)
+        super(numbers.presence || [])
```

Comment:
It was supposed to protect against empty string, but I think it is not possible right now to pass empty string here. Besides, this method is not even called due to the typo in the name. I will remove it.